devtools/chain/specs/single_node/chain-spec.toml

```diff
@@ -0,0 +1,62 @@
+#
+# Data of the genesis block.
+#
+[genesis]
+difficulty = "0x0"
```

Comment:
The difficulty in Axon is a fixed value, and it never changes. So this field should not appear in config.

Reply:
> So, should we remove them from `Header` too (in another PR)?
> 
> Ref: #888

Yes, so does `nonce`